simulator/examples/polyline-joints.rs

```diff
@@ -0,0 +1,97 @@
+use embedded_graphics::{
```

Comment:
You said that this example was mainly intended to debug the code, so should we remove this example in the PR and maybe add add a better one in a future PR? We still have `primitives-polyline` to demonstrate the basics.